charts/victoria-metrics-k8s-stack/Chart.yaml

```diff
@@ -2,14 +2,14 @@ apiVersion: v2
 name: victoria-metrics-k8s-stack
 description: Kubernetes monitoring on VictoriaMetrics stack. Includes VictoriaMetrics Operator, Grafana dashboards, ServiceScrapes and VMRules
 type: application
-version: 0.14.17
-appVersion: "1.90.0"
+version: 0.14.18
```

Comment:
I'm not sure about those version update

Reply:
helm charts update a little bit weird.

1) You have to bump version (minor or major). Usually it's minor for k8s stack
2) execute make package (will bundle helm chart archieve)
3) execute make index (will add bundle into index.yaml file).